packages/manager/apps/telecom/src/app/telecom/pack/meetings/xdsl-meetings.routing.js

```diff
@@ -1,6 +1,6 @@
 export default /* @ngInject */ ($stateProvider) => {
   $stateProvider.state('telecom.xdsl-meetings', {
-    url: '/xdsl/:serviceName/meetings',
+    url: '/meetings/:serviceName',
```

Comment:
Why change current url ? If serviceName is an xdsl then /xdsl/serviceName/meetings is fine, as we're getting meetings for a current service
Also no redirection handled so bookmarked url can be redirected ?

Reply:
@marie-j Sorry, I don't understand your second question.

Reply:
I change the url, because I think it's more valuable to point directly to meetings with the serviceName associated, this url will send to the customer in a mail when he need to take a new meeting for xDSL line creation. It's only accessible from a mail.

Reply:
What I meant is that the /xdsl/:serviceName/meetings should redirect to /meetings/:serviceName